samples/nextjs/src/lib/component-props.ts

```diff
@@ -7,14 +7,17 @@ import {
   RouteData,
 } from '@sitecore-jss/sitecore-jss-nextjs';
 
+export type StyleguideSitecoreContextValue = LayoutServiceContext & {
```

Comment:
@ambrauer Did it, but I'm not sure where can I put `SitecoreContextValue`... Let's leave it here, because it used by `StyleguideComponentWithContextProps`, and it's not related to `page-props`. And it's not a good idea to create a separate file for this type...